features/camel/src/main/feature/feature.xml

```diff
@@ -16,285 +16,561 @@
           xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
-    <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
-    <repository>mvn:ddf.features/cxf/${project.version}/xml/features</repository>
-    <repository>mvn:org.apache.karaf.features/enterprise/${karaf.version}/xml/features</repository>
+  <repository>mvn:ddf.features/cxf/${project.version}/xml/features</repository>
+  <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
+  <repository>mvn:org.apache.karaf.features/enterprise/${karaf.version}/xml/features</repository>
 
-    <!--START Camel Features-->
-    <!-- DO NOT EDIT
-        All feature definitions come from 3rd party feature file:
-        <repository>mvn:org.apache.camel.karaf/apache-camel/2.24.2/xml/features</repository>
+  <!--START Camel Features-->
+  <!-- DO NOT EDIT
+      All feature definitions come from 3rd party feature file:
+      <repository>mvn:org.apache.camel.karaf/apache-camel/3.4.4/xml/features</repository>
 
-        All version properties for these Camel features are defined in the platform-app pom.
+      All version properties for these Camel features are defined in the platform-app pom.
 
-        DDF Changes made:
-        - removed resolver="(obr)" from every feature
-        - added  to every feature
-        - replaced ${project.version} with ${camel.version}
-        - commented out xml-specs-api feature and replaced xml-specs-api with cxf-specs in camel-core feature
-        - added camel as feature prerequisite in camel-http to keep things ordered
-        - added cxf as feature prerequisite in camel to keep things ordered
-        - removed features that are not needed
-        - updated versions of qpid and proton versions and added netty bundles in camel-amqp
-        - replaced bundles in camel-jms
-        - replaced cxf-http-provider with cxf-http-jetty
-        - the versions for proton-j and qpid-jms-client used in camel-amqp need to be kept in sync with the versions used in artemis-amqp
-        - updated proton, qpid-jms, and netty versions to stay in sync with Camel 2.24.2 dependencies
+      DDF Changes made:
+      - replaced ${project.version} with ${camel.version}
+      - commented out xml-specs-api feature and replaced xml-specs-api with cxf-specs in camel-core feature
+      - removed features that are not needed
+      - replaced geronimo-jta_1.1_spec bundle with Karaf's "transaction" feature in a few places:
+        * camel-jms
```

Comment:
It may be better if DDF didn't maintain these and any other features it doesn't directly use and require downstream distribution to add them or use them directly from camel's feature file

Reply:
Agreed, I didn't check to see if these features were actually needed, I just kept what was there before. I'll remove everything that's no longer used

Reply:
Ok I went through and removed a bunch of features that weren't used